store/tikv/client.go

```diff
@@ -353,6 +354,13 @@ func (c *rpcClient) SendRequest(ctx context.Context, addr string, req *tikvrpc.R
 		return nil, errors.Trace(err)
 	}
 
+	metrics.TiKVRecycleConnDuration.Observe(float64(time.Since(start)))
+	if ctx.Value("isMeta") == true {
+		duration := time.Since(start)
+		startTs := ctx.Value(txnStartKey)
+		logutil.Logger(ctx).Info("load meta SendRequest", zap.Reflect("startTs", startTs), zap.Duration("recycle conn", duration))
```

Comment:
You mean a hold lock for a long time cause the latency here?

Reply:
Yes, but it's probably not. I just want to cover all possibilities.